arches/app/search/components/resource_type_filter.py

```diff
@@ -56,5 +57,11 @@ def append_dsl(
 
     def view_data(self):
         return {
-            "resources": get_resource_types_by_perm(self.request.user, "read_nodegroup")
+            "resources": list(
+                Graph.objects.filter(
```

Comment:
`get_resource_types_by_perm` returns `GraphModel`s . We could save a little overhead here if it's also possible to return those here instead of `Graph`s

Reply:
@aarongundel I wouldn't be surprised if the use of the proxy model here is the source of the performance issues that you've noticed.